lib/service/auth_service.dart

```diff
@@ -17,28 +17,32 @@ import 'package:autonomy_flutter/screen/bloc/subscription/subscription_state.dar
 import 'package:autonomy_flutter/screen/settings/subscription/upgrade_bloc.dart';
 import 'package:autonomy_flutter/screen/settings/subscription/upgrade_state.dart';
 import 'package:autonomy_flutter/service/configuration_service.dart';
+import 'package:autonomy_flutter/service/navigation_service.dart';
+import 'package:autonomy_flutter/service/passkey_service.dart';
 import 'package:autonomy_flutter/util/exception.dart';
 import 'package:autonomy_flutter/util/log.dart';
+import 'package:autonomy_flutter/util/user_account_channel.dart';
 import 'package:easy_localization/easy_localization.dart';
 
 class AuthService {
   final IAPApi _authApi;
   final UserApi _userApi;
   final ConfigurationService _configurationService;
   JWT? _jwt;
+  final UserAccountChannel _userAccountChannel;
 
-  AuthService(
-    this._authApi,
-    this._userApi,
-    this._configurationService,
-  );
+  AuthService(this._authApi,
+      this._userApi,
+      this._configurationService,
+      this._userAccountChannel,);
 
-  void reset() {
-    setAuthToken(null);
+  Future<void> reset() async {
+    await setAuthToken(null);
   }
 
-  Future<JWT> refreshJWT({String? receiptData}) async {
-    final refreshToken = _jwt?.refreshToken;
+  Future<JWT> _refreshJWT({String? receiptData}) async {
+    final jwt = _jwt;
+    final refreshToken = jwt?.refreshToken;
```

Comment:
`final refreshToken = _jwt?.refreshToken;`

Why we should declare `jwt` variable?